qs.gemspec

```diff
@@ -21,9 +21,9 @@ Gem::Specification.new do |gem|
   gem.add_dependency("dat-worker-pool", ["~> 0.4"])
   gem.add_dependency("hella-redis",     ["~> 0.2"])
   gem.add_dependency("ns-options",      ["~> 1.1"])
-  gem.add_dependency("oj",              ["~> 2.11"])
   gem.add_dependency("SystemTimer",     ["~> 1.2"])
 
   gem.add_development_dependency("assert", ["~> 2.12"])
+  gem.add_development_dependency("json",   ["~> 1.8"])
```

Comment:
@jcredding I'd expect this Gemfile to bring this in though, since we still test/bench in 1.8.7.